### DISCUSSION TITLE
Possibilità di inserire il verbale dello scrutinio in bacheca

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#267] [Idee e proposte]
Buonasera, sarebbe possibile nel prossimi aggiornamenti poter inserire nella sezione bacheca l'opzione, oltre compiti assegnati e verifiche, di poter inserire documenti che non siano Relazioni, 15 Maggio, Programmi e Piani di lavoro? Ad esempio io dovevo inserire il verbale dello scrutinio finale per la classe in cui sono coordinatore ma sono stato costretto a inserirlo in Avvisi non avendo altri modi.

**Comment by trinko** [comment]
Si, l'idea è quella: in futuro si potranno inserire documenti di vario tipo, da parte dello staff, nella Bacheca > Documenti.

Attualmente anche noi utilizziamo gli avvisi per casi analoghi.
